### PR TITLE
docs(grid): Updated remote virtualization topic - added sorting and filtering

### PR DESCRIPTION
**Pull request by sboykova**
Related to IgniteUI/igniteui-angular-samples#272

### CODE REVIEW

**Comment by yoshimis** [comment]
code part in jp md file needs to be updated.

**Comment by kdinev** [comment]
@sboykova Could you take care of the code snippet update in the JP version?

File: jp/components/grid_virtualization.md [hunk 1]
```diff
@@ -76,34 +82,16 @@ public dataLoading(evt) {
...
-    }
-    return `${this.url}${qS}`;
-}
-```
+<div class="sample-container loading" style="height:550px">
+    <iframe id="grid-remote-filtering-iframe" src='{environment:demosBaseUrl}/grid-remote-filtering' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="grid-remote-filtering-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
```
Comment (ig-ealbert) [inline jp/components/grid_virtualization.md]: @yoshimis Please translate the button text.

**Comment by kdinev** [comment]
@yoshimis Once you apply the change requested by @ig-ealbert I will merge the PR.

**Pull request merged**